collection/collector.go

```diff
@@ -8,10 +8,13 @@ import (
 	pb "github.com/odpf/raccoon/proto"
 )
 
+type AckFunc func(err error)
```

Comment:
(suggestion) nit: we can rename to 

```go
// AckEventFunc acknowledges to the client
type AckEventFunc func(err error)
```

Reply:
I don't think this is required as the type itself is a function.